src/page/Variant.tsx

```diff
@@ -29,6 +30,8 @@ class Variant extends React.Component<IVariantProps> {
         win.props = props;
     }
 
+    @observable shouldOpenWarningWindow = true;
```

Comment:
this variable name makes me feel like the warning window should be open by default. maybe rename it to something like `warningWindowClosedByUser` and set `false` as the default value?